framework/src/main/java/org/tron/common/logsfilter/capsule/TransactionLogTriggerCapsule.java

```diff
@@ -156,6 +163,90 @@ public TransactionLogTriggerCapsule(TransactionCapsule trxCapsule, BlockCapsule
                     StringUtil.encode58Check(createSmartContract.getOwnerAddress().toByteArray()));
               }
               break;
+            case UnfreezeBalanceContract:
+              UnfreezeBalanceContract unfreezeBalanceContract = contractParameter
+                  .unpack(UnfreezeBalanceContract.class);
+
+              if (Objects.nonNull(unfreezeBalanceContract)) {
+                transactionLogTrigger.setFromAddress(StringUtil
+                      .encode58Check(unfreezeBalanceContract.getOwnerAddress().toByteArray()));
+                if (Objects.nonNull(unfreezeBalanceContract.getReceiverAddress())) {
+                  transactionLogTrigger.setToAddress(StringUtil
+                        .encode58Check(unfreezeBalanceContract.getReceiverAddress().toByteArray()));
+                }
+              }
+              break;
+            case FreezeBalanceV2Contract:
+              FreezeBalanceV2Contract freezeBalanceV2Contract = contractParameter
+                  .unpack(FreezeBalanceV2Contract.class);
+
+              if (Objects.nonNull(freezeBalanceV2Contract)) {
+                transactionLogTrigger.setFromAddress(StringUtil
+                      .encode58Check(freezeBalanceV2Contract.getOwnerAddress().toByteArray()));
+                transactionLogTrigger.setAssetName("trx");
+                transactionLogTrigger.setAssetAmount(freezeBalanceV2Contract.getFrozenBalance());
+              }
+              break;
+            case UnfreezeBalanceV2Contract:
+              UnfreezeBalanceV2Contract unfreezeBalanceV2Contract = contractParameter
+                  .unpack(UnfreezeBalanceV2Contract.class);
+
+              if (Objects.nonNull(unfreezeBalanceV2Contract)) {
+                transactionLogTrigger.setFromAddress(StringUtil
+                      .encode58Check(unfreezeBalanceV2Contract.getOwnerAddress().toByteArray()));
+                transactionLogTrigger.setAssetName("trx");
+                transactionLogTrigger.setAssetAmount(
+                      unfreezeBalanceV2Contract.getUnfreezeBalance());
+              }
+              break;
+            case WithdrawExpireUnfreezeContract:
+              WithdrawExpireUnfreezeContract withdrawExpireUnfreezeContract = contractParameter
+                  .unpack(WithdrawExpireUnfreezeContract.class);
+
+              if (Objects.nonNull(withdrawExpireUnfreezeContract)) {
+                transactionLogTrigger.setFromAddress(StringUtil.encode58Check(
+                      withdrawExpireUnfreezeContract.getOwnerAddress().toByteArray()));
+                transactionLogTrigger.setAssetAmount(transactionInfo.getWithdrawExpireAmount());
```

Comment:
transactionLogTrigger.setAssetName("trx");

Reply:
OK, I get it